### PR TITLE
Added Swahili(sw) locale

### PR DESCRIPTION
**Pull request by Paulmburu**

### CODE REVIEW

**Comment by rugk** [comment]
Hi @Paulmburu,
thanks for your first contribution to this project! :tada: :+1: 🏅
I hope you'll like this project and enjoy hacking on it… :smiley:

Please subscribe to #138 to get notified for new updates on translations etc. :smiley:

**Comment by rugk** [comment]
@Paulmburu Are you still interested in completing this? If not, that's no problem, just add a notice, so I know what the current status of this PR is.

**Comment by Paulmburu** [comment]
Yeah, i will get it done before the day ends

**Comment by Paulmburu** [comment]
Swahili locale indentation adjusted to two spaces

File: src/_locales/sw/messages.json [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+    "placeholders": {
+      "translators": {
+        "content": "$1",
+        "example": "<a href=\"https://github.com/rugk/\">@rugk</a>"
+      }
+    },
+    "hash": "a6f5deecd624d5f8cd58350c72dd8fdf"
+  },
+  "translatorLink": {
+    "message": "https://github.com/rugk",
```
Comment (rugk) [inline src/_locales/sw/messages.json]: I guess you want to credit yourself here.

File: src/_locales/sw/messages.json [hunk 2]
```diff
@@ -0,0 +1,502 @@
...
+    "description": "The link to the translator's GitHub profile.",
+    "hash": "4775fb2f88fd33d02b871e5021dec3e9"
+  },
+  "translatorUsername": {
+    "message": "rugk",
```
Comment (rugk) [inline src/_locales/sw/messages.json]: I guess you want to credit yourself here.

**Comment by Paulmburu** [comment]
update translators link and resume

File: src/_locales/sw/messages.json [hunk 3]
```diff
@@ -0,0 +1,500 @@
...
+  "translatorLink": {
+    "message": "https://github.com/Paulmburu",
+    "description": "The link to the translator's GitHub profile."
+  },
+  "translatorUsername": {
+    "message": "Paulmburu",
+    "description": "The translator's GitHub username."
+  },
+  "contributorsThanks": {
+    "message": "Pia shukrani kwa $ WAKUWA WA $ $.",
```
Comment (rugk) [inline src/_locales/sw/messages.json]: I guess you mistakenly translated the part inside of the `$$` signs too. Please do not do that. That is a placeholder, where other text is inserted.

See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translations for how these placeholders are translated, especially [the official doc](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales).

File: src/_locales/sw/messages.json [hunk 4]
```diff
@@ -0,0 +1,500 @@
...
+    "description": "The heading/grouping title shown for the miscellaneous setting of the QR code.",
+    "hash": "6394c364dd4afc224c653644259bc4ea"
+  },
+  "optionQrQuietZone": {
+    "message": "Ukanda wa utulivu wa QR:",
+    "description": "This is an option shown in the add-on settings. It's about the area around the QR code.",
+    "hash": "a11f6e534cf14d30bccdb359489862b9"
+  },
+  "optionQrQuietZoneStatusSingular": {
+    "message": "($ INTEGER $ mraba)",
```
Comment (rugk) [inline src/_locales/sw/messages.json]:
```suggestion
    "message": "($INTEGER$ mraba)",
```

do not add spaces

**Comment by rugk** [comment]
@Paulmburu If you need any pointers/help or have questions, feel free to ask.

**Comment by Paulmburu** [comment]
On it, my apologies for the delay

**Comment by rugk** [comment]
No problem. Did you test this? Can this be merged?

Note you may also be advised to check the diff (difference) between [the current up-to-date English template](https://github.com/rugk/offline-qr-code/blob/master/src/_locales/en/messages.json) and your version. Just as it is so outdated, it could miss some strings or so. [The online translator](https://lusito.github.io/web-ext-translator/) should do this, if you enter/use your repo URL.

To see the changes, you may however need to [merge the changes from upstream into your fork](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork).

**Comment by Paulmburu** [comment]
let me know of any arising issues

**Comment by rugk** [comment]
Okay, thanks again. It's merged.

As said, please subscribe to #138 to get notified for new updates on translations etc. :smiley:

Also notice there are more things to translate like the add-on listings on AMO (addons.mozilla.org) etc. [See the contributing guide for more information.](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translation-of-add-on-listings)

**Pull request merged**